rosdep/base.yaml

```diff
@@ -2988,7 +2991,7 @@ libgconf2:
 libgdal-dev:
   debian: [libgdal-dev]
   nixos: [gdal]
-  opensuse: [gdal]
+  opensuse: [gdal-devel]
```

Comment:
I found a reference to the devel version in the spec file here too: https://build.opensuse.org/package/view_file/openSUSE:Leap:15.2/gdal/gdal.spec?expand=1

Reply:
I did the same thing for glew
https://build.opensuse.org/package/binaries/openSUSE:Leap:15.2/glew/standard